data/bedrock/latest/proto.yml

```diff
@@ -3726,3 +3732,41 @@ packet_feature_registry:
    features: []varint
       name: string
       options: string
+
+packet_server_stats:
+   !id: 0xc0
+   !bound: client
+   server_time: lf32
+   network_time: lf32
+
+packet_request_network_settings:
+   !id: 0xc1
+   !bound: server
+   client_protocol: i32
+
+packet_game_test_request:
+   !id: 0xc2
+   # MaxTestsPerBatch ...
+   max_tests_per_batch: varint
+   # Repetitions represents the amount of times the test will be run.
+   repetitions: varint
+   # Rotation represents the rotation of the test. It is one of the constants above.
+   rotation: u8
```

Comment:
we can add the constants in https://github.com/Sandertv/gophertunnel/pull/146/files#diff-7da5dae79269c9351484664f4b134a04deafc18b605327b4ee649659907d078cR5 for clarity that this is not an analog field

Reply:
Done and done. Good catch!

Re game_test_request.rotation: 

1) okay... i guess that works. I suppose they would have to step up to 16-bit if they wanted 360 degrees.
2) Kept them as indexed integers. Would strings be better? or perhaps qualified strings (`"rotation0"`)?:
3) 0... and 360?

Reply:
Yeah, using numbers in the enum isn't good because that can create data type collisions. As users can specify the string enum representation *or* specify a numeric value for these fields, this means ambiguity when serializing in protodef. I've corrected them to be strings.